src/main/java/net/sf/jabref/MetaData.java

```diff
@@ -30,9 +30,10 @@
 
     public static final String META_FLAG = "jabref-meta: ";
     public static final String SAVE_ORDER_CONFIG = "saveOrderConfig";
+    public static final String SAVE_ACTIONS = "saveActions";
```

Comment:
You are right. In that case it is probably best to leave the name as it is and to avoid the creation of potential migration problems.